src/components/aticles-list.jsx

```diff
@@ -1,30 +1,35 @@
+import { ArticleCard } from "./article-card";
+
 export function ListArticle({ articles }) {
   return (
-    <div>
-      <ul>
-        {articles.map(
-          ({
+    <>
+      {articles.map(
+        (
+          {
             article_id,
             title,
             topic,
             author,
             created_at,
             votes,
             comment_count,
-          }) => {
-            return (
-              <li className="article-list" key={article_id}>
-                <h3>{title}</h3>
-                <h4>Topic: {topic}</h4>
-                <h4>Author: {author}</h4>
-                <h4>Date: {created_at}</h4>
-                <h4>Votes: {votes}</h4>
-                <h4>Comments: {comment_count}</h4>
-              </li>
-            );
-          }
-        )}
-      </ul>
-    </div>
+          },
+          index
+        ) => {
+          return (
+            <ArticleCard
+              key={index}
+              article_id={article_id}
```

Comment:
Using the `index` from a map as the `key` prop will satisfy React's warnings, but it can effect performance of your app a little. The good news is that you have a perfectly good unique value in `article_id` that you can use instead!

Reply:
Thanks for the feedback.. I will make those changes.